### ISSUE TITLE
result: May contain several patches

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#741]
blocked by https://github.com/coala-analyzer/coala/issues/487

**Comment by sils1297** [comment]
Nope, I don't see why this needs #487 first... though they are related. One at a time please.

**Comment by sils1297** [comment]
This was already done. Apparently. At least the result has a diff dict.
